### PR TITLE
RateLimit documentation fixes July 2020

### PR DESCRIPTION
**Pull request by akira-kurogane**

### CODE REVIEW

File: source/rate-limit.rst [hunk 1]
```diff
@@ -32,6 +32,45 @@ With rate limiting you can collect profiling data for all database operations
...
+     - yes
+     - no
+   * - Example value of option
+     - 0.02
+     - 50
+     
+rateLimit is a better way to have continual profiling for monitoring or live
+analysis purposes. sampleRate requires setting slowOpsThresholdMs to zero if
+you wanted to sample all types of ops, and has an effect on the log file
+which may decrease or increase log volume.
```
Comment (nastena1606) [inline source/rate-limit.rst]:
```suggestion
``rateLimit`` is a better way to have continuous profiling for monitoring or live
analysis purposes. ``sampleRate`` requires setting slowOpsThresholdMs to zero if
you want to sample all types of operations. ``sampleRate`` has an effect on the log file
which may either decrease or increase the log volume.
```

File: source/rate-limit.rst [hunk 2]
```diff
@@ -32,6 +32,45 @@ With rate limiting you can collect profiling data for all database operations
 and reduce overhead by sampling queries.
 Slow queries ignore rate limiting and are always collected by the profiler.
 
+Comparing to "sampleRate" option
+================================
+
+The sampleRate option (= slowOpSampleRate_ config file option) is a similar
+concept to rateLimit. But it works at different profile level, completely 
+ignores ops faster than slowOpsThresholdMs (a.k.a. slowMs), and affects the 
+log file printing too.
```
Comment (nastena1606) [inline source/rate-limit.rst]:
```suggestion
The ``sampleRate`` option (= slowOpSampleRate_ config file option) is a similar
concept to ``rateLimit``. But it works at different profile level, completely 
ignores ops faster than ``slowOpsThresholdMs`` (a.k.a. ``slowMs``), and affects the 
log file printing too. 
```

**Review by nastena1606 — COMMENTED** [review]
Akira I have made some edits in formatting and tried to split long sentences into shorter ones.

File: source/rate-limit.rst (hunk 2 above)
Comment (akira-kurogane) [reply source/rate-limit.rst]: lgtm

File: source/rate-limit.rst (hunk 1 above)
Comment (akira-kurogane) [reply source/rate-limit.rst]: The new "It" at the start of the last sentence becomes a bit unclear - is it rateLimit or sampleRate.

Suggest that "sampleRate" be repeated if ", and" is being removed.

**Review by igorsol — APPROVED** [review]

File: source/rate-limit.rst (hunk 1 above)
Comment (nastena1606) [reply source/rate-limit.rst]: Agree

**Review by nastena1606 — APPROVED** [review]

**Pull request merged**